src/test/java/org/folio/rest/impl/OrdersImplTest.java

```diff
@@ -2513,6 +2519,17 @@ public void testPostOrdersLinesByIdPoLineWithoutId(TestContext ctx) throws IOExc
     ctx.assertEquals(ErrorCodes.MISSING_ORDER_ID_IN_POL.getCode(), resp.getErrors().get(0).getCode());
   }
 
+  @Test
+  public void testPostOrdersWithMissingVendorId(TestContext ctx) throws IOException {
+    logger.info("=== Test Post Order with missing Vendor Id ===");
+
+    Errors resp = verifyPostResponse(COMPOSITE_ORDERS_PATH, getMockData(ORDER_WITHOUT_VENDOR_ID),
+      prepareHeaders(NON_EXIST_CONFIG_X_OKAPI_TENANT), APPLICATION_JSON, 422).as(Errors.class);   
+    
+    ctx.assertEquals(1, resp.getErrors().size());
+    ctx.assertEquals(ErrorCodes.VENDOR_ID_REQUIRED.getCode(), VENDOR_ID_REQUIRED);
```

Comment:
I think you'll wind up with something like this:
```
{
	"errors": [{
		"message": "may not be null",
		"type": "1",
		"code": "-1",
		"parameters": [{
			"key": "orderFormat",
			"value": "null"
		}]
	}]
}
```
Only with key: `vendor` instead of `orderFormat`

Reply:
Yes...